src/common/http.test.ts

```diff
@@ -16,13 +17,80 @@ import {
   fetchProviderInfo,
   fetchProviders,
   getServicesUrl,
+  SuperfaceClient,
 } from '../common/http';
 import { mockResponse } from '../test/utils';
 import { DEFAULT_PROFILE_VERSION_STR } from './document';
+import { loadNetrc } from './netrc';
+import { ProfileId } from './profile';
+
+jest.mock('./netrc');
+
+describe('SuperfaceClient', () => {
+  describe('getClinet', () => {
+    let sfClient: typeof SuperfaceClient;
+
+    beforeEach(async () => {
+      sfClient = (await import('../common/http')).SuperfaceClient;
+    });
+
+    afterEach(() => {
+      jest.resetModules();
+    });
+
+    it('should return clinet', async () => {
+      const mockNetRcRecord = {
+        baseUrl: 'baseUrl',
+        refreshToken: 'RT',
+      };
+      mocked(loadNetrc).mockReturnValue(mockNetRcRecord);
+      const clinet = sfClient.getClient();
```

Comment:
Got some of these bellow as well 😄 

```suggestion
      const client = sfClient.getClient();
```